### DISCUSSION TITLE
Need help

### DISCUSSION THREAD

**Discussion by nucleusgenius9** [postalserver/postal#2230] [Help with using Postal] [answered]
I can't solve problems with the postal setup (see screenshot 1)

![1](https://user-images.githubusercontent.com/118116015/201525647-0b98bc42-46e9-45ed-9c07-d28836b8e3f4.jpg)

there is a required line in the dns settings, but it still does not work
![2](https://user-images.githubusercontent.com/118116015/201525787-661a9eaf-87c4-42e4-8971-3f45ce1c72ea.jpg)

**Comment by willpower232** [comment, marked as answer]
Your problem is that you haven't removed the `example.com` 's from the config file.

If you have, then you haven't restarted Postal to apply the updated config file. Postal is actually looking for example.com currently which is not correct and usually means the config file needs updating.

**Reply by nucleusgenius9** [reply]
Thanks for the answe.
